src/aat/resources/features/F-037/S-577.td.json

```diff
@@ -0,0 +1,156 @@
+{
+  "title": "must return successfully all and only the respective fields with READ access for a Case Worker and a Solicitor",
+  "_extends_": "S-577_Update_Case_For_Case_Worker_Data_Base",
+  "_guid_": "S-577",
+  "specs": [
+    "an active Case Worker profile in CCD with a specific variation of ACLs on a case type",
+    "an active solicitor profile with another specific variation of ACLs on the same case type",
+    "is prepared on behalf of the user creating the case above",
+    "contains updated values for DocumentField2"
+  ],
+  "user": {
+    "username": "befta.caseworker.2.solicitor.2@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "body": {
+      "data": {
+        "DocumentField2":{
+          "document_url":"http://dm-store:8080/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
+          "document_binary_url":"http://dm-store:8080/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943/binary",
+          "document_filename":"Elastic Search test Case.png --> updated by Solicitor 2"
+        }
+      }
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 200,
+    "responseMessage": "OK",
+    "headers": {
+      "_extends_": "Common_Response_Headers"
+    },
+    "body" : {
+      "_links" : {
+        "self" : {
+          "href" : "[[ANY]]"
+        }
+      },
+      "id" : "[[ANY]]",
+      "jurisdiction" : "BEFTA_JURISDICTION_2",
+      "case_type" : "BEFTA_CASETYPE_2_1",
+      "created_on" : "[[ANY]]",
+      "last_modified_on" : "[[ANY]]",
+      "state" : "TODO",
+      "security_classification" : "PUBLIC",
+      "data" : {
```

Comment:
could extend `Befta_Default_Full_Case_Data`

Reply:
Good catch, thanks. Updated.

Reply:
Unfortunately it cannot be done as this case is update for documentField2, so I had to revert this inheritance.

Reply:
it could still use inheritance, just keep the updated DocumentField2 here which will override the inherited value

Reply:
that's right, it is working that way.